CreeDictionary/DatabaseManager/xml_importer.py

```diff
@@ -462,22 +463,14 @@ def import_xmls(dir_name: Path, multi_processing: int = 1, verbose=True):
         db_lemma = None
         _, _, xml_lc = xml_lemma_pos_lcs[0]
 
-        # for use in building/merging definition
-        xml_lemmas = {t[0] for t in xml_lemma_pos_lcs}
-
-        xml_lemma_to_definition_src_dicts: Dict[str, List[Dict[str, Set[str]]]] = {
-            xml_lemma: [
-                xml_lemma_pos_lc_to_str_definitions[(xml_lemma,) + pos_lc_tuple]
-                for pos_lc_tuple in xml_lemma_to_pos_lc[xml_lemma]
-            ]
-            for xml_lemma in xml_lemmas
-        }
-
```

Comment:
A major task below is to find definitions from entries in the xml and associate them with machine generated wordforms.

`xml_lemma_to_definition_src_dicts` here is built and used in the process. The old way negalects pos/lc in the association process and uses the same definitions for homographs (the pipon case for e.g.).

Also this is mad comprehension to an extend of unneccessary. Thanks god it's finally deleted.

Reply:
Deleting code is always great!